src/lib/__tests__/server-utilities.test.js

```diff
@@ -72,3 +72,54 @@ describe('compressJPEG', () => {
     });
   });
 });
+
+describe('deleteFile', () => {
+  const source = './src/lib/__tests__/assets/copy-test.jpg';
+  const target = './tmp/copy-test.txt';
+
+  beforeAll(() => {
+    if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+    return copyFile(source, target);
+  });
+
+  describe('successfully deletes file', () => {
+    expect.assertions(1);
+    if (fs.existsSync(target)) {
+      deleteFile(target).then(() => {
+        expect(fs.existsSync(target)).not.toBe(true);
+      });
+    }
+  });
+});
+
+describe('copyFile', () => {
+  const source = './src/lib/__tests__/assets/copy-test.txt';
+  const target = './tmp/copy-test.txt';
+
+  beforeAll(() => {
+    if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+    return copyFile(source, target);
+  });
+
+  it('copies file to new directory', () => {
+    expect(fs.existsSync(target)).toBe(true);
+  });
+
+  it("does't remove the old file", () => {
+    expect(fs.existsSync(source)).toBe(true);
+  });
+
+  it("doesn't copy a file that doesn't exist", () => {
+    expect.assertions(1);
+    const nonexistentSource = './src/lib/__tests__/assets/nonexistent-file.test';
+    const nonexistentTarget = './tmp/nonexistent-file.test';
+    return copyFile(nonexistentSource, nonexistentTarget).catch((error) => {
```

Comment:
Hmm I'm pretty sure there's a better way to do the `expect` stuff here, I gotta get back to orientation pretty soon, I might just check up on it now, and comment later, off the top of my head you could definitely use `.instaceOf` or something like that, but what I really want is something `toThrow` which of course is for a sync function though... so let me just look at it, or if you find something of course also feel free to comment / edit

Reply:
there we go https://facebook.github.io/jest/docs/en/expect.html#rejects